Los Angeles County Infant Supplement and Expectant Parent Payment

PavelMakarchuk opened the pull request:
Fixes #5319

MaxGhenis reviewed (changes requested):
Shouldn't we make all these monthly?

Could you cite the WIC (legal code) pointed to in the pdf doc? In particular I'm unclear on the definition of infant here.

Also check some typos in descriptions

PavelMakarchuk commented:
> Shouldn't we make all these monthly?
> 
> Could you cite the WIC (legal code) pointed to in the pdf doc? In particular I'm unclear on the definition of infant here.
> 
> Also check some typos in descriptions

Refactored to monthly and added the infant definition

MaxGhenis commented on policyengine_us/parameters/gov/local/ca/la/dss/expectant_parent_payment/amount.yaml:
```diff
@@ -0,0 +1,12 @@
+description: The California Department of Social Services provides the following Expectant Parent Payment amount, based on pregnancy months. 
```
`based on` usually for breakdowns
```suggestion
description: The California Department of Social Services provides the following Expectant Parent Payment. 
```

MaxGhenis commented on policyengine_us/parameters/gov/states/ca/infant/age_limit.yaml:
```diff
@@ -0,0 +1,11 @@
+description: California defines infants as children below this age threshold.
```
the formula does `<=` though, and the test says 1 is eligible

MaxGhenis commented on policyengine_us/tests/policy/baseline/gov/local/ca/la/dss/infant_supplement/ca_la_infant_supplement.yaml:
```diff
@@ -0,0 +1,64 @@
+- name: Eligble household with one infant, not in group home
```
```suggestion
- name: Eligible household with one infant, not in group home
```

MaxGhenis commented on policyengine_us/variables/gov/local/ca/la/dss/expectant_parent_payment/ca_la_expectant_parent_payment.py:
```diff
@@ -0,0 +1,13 @@
+    definition_period = MONTH
+    label = "Los Angeles County expectant parent payment"
+    defined_for = "ca_la_expectant_parent_payment_eligible"
+
```
right? or [] around?
```suggestion
    adds = "gov.local.ca.la.dss.expectant_parent_payment.amount"
```

PavelMakarchuk replied:
Forgot about the [] - works here

MaxGhenis commented on policyengine_us/variables/gov/local/ca/la/dss/infant_supplement/ca_la_infant_supplement.py:
```diff
@@ -0,0 +1,27 @@
+        eligible_person = person(
+            "ca_la_infant_supplement_eligible_person", period
+        )
+        eligble = household.any(eligible_person)
```
```suggestion
        eligible = household.any(eligible_person)
```

PavelMakarchuk commented:
> please point to the WIC where relevant https://leginfo.legislature.ca.gov/faces/codes_displaySection.xhtml?lawCode=WIC&sectionNum=11465.
> 
> I think some of this is actually dependent on other policy parameters around foster care

If referring to the amounts - I am not sure which other program WIC refers to - I have added the legal code to the relevant sections

MaxGhenis commented on policyengine_us/variables/gov/local/ca/la/dss/infant_supplement/ca_la_infant_supplement_eligible_infant.py:
```diff
@@ -0,0 +1,15 @@
+
+    def formula(person, period, parameters):
+        p = parameters(period).gov.local.ca
+        age = person("age", period)
```
```suggestion
```

MaxGhenis commented on policyengine_us/variables/gov/local/ca/la/dss/infant_supplement/ca_la_infant_supplement_eligible_infant.py:
```diff
@@ -0,0 +1,15 @@
+    defined_for = "in_la"
+
+    def formula(person, period, parameters):
+        p = parameters(period).gov.local.ca
```
```suggestion
```
